tetris.py

```diff
@@ -296,6 +319,35 @@ def get_block_shape(self):
 
         return coords[self.rotate_counter % len(coords)]
 
+    def get_upcoming_block_shape(self):
+        if self.upcoming_block_letter == 'I':
+            self.upcoming_block_center = (2, 0)
+        else:
+            self.upcoming_block_center = (1, 1)
+        (x, y) = self.upcoming_block_center
```

Comment:
There's no need to store the center to `self`. If you use a local variable instead, the name can be shorter too.

```suggestion
        if self.upcoming_block_letter == 'I':
            center = (2, 0)
        else:
            center = (1, 1)
        (x, y) = center
```